man/dracut.cmdline.7.asc

```diff
@@ -1159,9 +1159,9 @@ device LABEL, and _<uuid>_ is the device UUID.
 * _none_ (the word itself) specifies that no overlay will be used, such as when
 an uncompressed, writable live root filesystem is available.
 +
-If a persistent overlay __is detected__ at the standard LiveOS path, the
-overlay & overlay type detected, whether Device-mapper or OverlayFS, will be
-used.
+If a persistent overlay __is detected__ at the standard LiveOS path,
+and rd.live.overlay.overlayfs is not set to 1, the overlay & overlay
+type detected, whether Device-mapper or OverlayFS, will be used.
```

Comment:
```suggestion
If a persistent overlay __is detected__ at the standard LiveOS path,
and rd.live.overlay.overlayfs is not set to 1, the overlay type (whether
Device-mapper or OverlayFS) will be detected and it will be used.
```